pkg/azure/actuator_test.go

```diff
@@ -463,9 +509,31 @@ func TestActuator(t *testing.T) {
 		},
 		{
 			name:        "Mint annotation",
-			expectedErr: fmt.Errorf("error determining whether a credentials update is needed"),
+			expectedErr: fmt.Errorf("unexpected value or missing cloudcredential.openshift.io/mode annotation on admin credentials Secret"),
```

Comment:
Note to reviewers: this change is a result of erroring earlier in the process as a result of checking for the cloudcredentials mode.